plugins/com.google.cloud.tools.eclipse.appengine.compat.cte13/src/com/google/cloud/tools/eclipse/appengine/compat/cte13/RequiresUpdatePropertyTester.java

```diff
@@ -0,0 +1,13 @@
+
+package com.google.cloud.tools.eclipse.appengine.compat.cte13;
+
+import org.eclipse.core.expressions.PropertyTester;
+import org.eclipse.core.resources.IProject;
+
+public class RequiresUpdatePropertyTester extends PropertyTester {
+  @Override
+  public boolean test(Object receiver, String property, Object[] args, Object expectedValue) {
+    return receiver instanceof IProject
+        && CloudToolsEclipseProjectUpdater.hasOldContainers((IProject) receiver);
```

Comment:
& not has Maven Nature

Reply:
We don't have any containers in Maven projects.